identity-enabler/deviceId-mobile-app/src/models/types/Window.d.ts

```diff
@@ -0,0 +1,3 @@
+declare interface Window {
+    onScan: (decodedText: string) => Promise<void>;
+}
```

Comment:
is this really necessary, probably not. that can be fixed by just disabling eslint on window as Any line